mlflow/CVE-2023-6977/README.md

```diff
@@ -0,0 +1,11 @@
+# MLflow CVE-2023-6977
+
+This directory contains the deployment config for MLflow instances vulnerable and fixed to CVE-2023-6977. MLflow versions below 2.9.2 are vulnerable to that arbitrary file read vulnerability.
+
+The deployed service listens on port `5000` after the docker completes its job.
+
+## Vulnerable version
```

Comment:
Isn't that the fixed version rather? (v2.10.0 was fixed per my understanding, but I might be wrong)